doc/user/cluster_tls.md

```diff
@@ -0,0 +1,73 @@
+# Cluster TLS Guide
+
+Cluster TLS policy is configured on a per-cluster basis via the TPR spec provided to etcd-operator.
+
+```yaml
+apiVersion: "etcd.coreos.com/v1beta1"
+kind: "Cluster"
+metadata:
+  name: "example-etcd-cluster"
+spec:
+  ...
+  clusterTLS:
+    ....
+```
+
+For a review etcd's TLS support and requirements, please read the relevant section from the [operations guide](https://coreos.com/etcd/docs/latest/op-guide/security.html).
+
+
+## Static Cluster TLS Policy
+
+```yaml
+apiVersion: "etcd.coreos.com/v1beta1"
+kind: "Cluster"
+metadata:
+  name: "example-etcd-cluster"
+ spec:
+  ...
+  clusterTLS:
+    static:
+      serverSecretName: server-tls-secret
+      clientSecretName: client-tls-secret
+
+```
+
+* **serverSecretName**: contains pem-encoded private keys and x509 certificates needed by the etcd server.
+
+  etcd-operator will mount this secret at `/etc/etcd-operator/server-tls` for each etcd member pod in the cluster.
+
+  The server TLS assets are expected to conform to the following structure:
+
+  ```text
+  /etc/etcd-operator/server-tls/
+
+       server-client-crt.pem
+       server-client-key.pem
+       ca-client-crt.pem
+
+       server-peer-crt.pem
+       server-peer-key.pem
+       ca-peer-crt.pem
+  ```
+
+  How these files are used by the etcd server is outlined in the [security flags section of the etcd admin guide](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#security-flags).
+
+* **clientSecretName**: contains pem-encoded private-key and x509 certificates needed to access etcd client interface. This identity is used by `etcd-operator` to access the cluster's client interface.
+
+  etcd-operator will mount this secret at `/etc/etcd-operator/client-tls` for every pod that needs to access the etcd cluster's client interface. (eg: backup pod)
+
+  The client TLS assets are expected to conform to the following structure:
+
+  ```text
+  /etc/etcd-operator/client-tls/
+
+	client-crt.pem
+	client-key.pem
+    ca-crt.pem
+
+  ```
+
+  These files represent the `--cert-file`,`--key-file`, and `--ca-file` arguments (respectively) to `etcdctl`.
+
+  It good to note that etcd-operator will also use these TLS secrets to access the etcd cluster, but will not mount the secret as a filesystem. This is so etcd-operator does not have to change it's own pod definition at runtime :smiley: .
```

Comment:
> I don't understand what you are trying to prove here?

I'm commenting that `etcd-operator` pod will consume the contents of the client tls secret, but will not mount the secret as a volume.